game/src/game/player.rs

```diff
@@ -29,26 +60,66 @@ pub struct PlayerGfxBundle {
     sprite: SpriteSheetBundle,
     animation: SpriteAnimationBundle,
 }
-
+//where is this added/spawned, currently debug spawned only
 #[derive(Component, Default)]
 pub struct PlayerBlueprint;
 
-#[derive(Component, Default)]
-pub struct PlayerGent;
+#[derive(Component)]
+pub struct PlayerGent {
+    e_gfx: Entity,
+}
 
 #[derive(Component, Default)]
 pub struct PlayerGfx;
 
+#[derive(Actionlike, PartialEq, Eq, Clone, Copy, Hash, Debug, Reflect)]
+pub enum PlayerAction {
+    MoveLeft,
+    MoveRight,
+    Jump,
+}
+
+//would be nice to have a function to check direction vector
```

Comment:
LWIM has directional inputs for this reason. Instead of having two actions MoveLeft and MoveRight, you could have a single Move action which is treated as an axis. It can be analog when playing with a controller and set to -1.0/1.0 when playing with a keyboard ("virtual axis"). You can even do this in two dimensions if you also want to have vertical movement ("dual axis"). Check the LWIM documentation.

That would be a much cleaner way to do our input.

Reply:
Okay looking into it. I think I would prefer `VirtualDPad` which looks like the same thing but clamped to -1. 0. 1. values. I think it will make it harder to tune jump control if we use true analog for jumping. If we want to switch we can always change later.

Reply:
hmmm maybe not, looks like `VirtualDPad` has to be a `DualAxis` and I *think* I would prefer single.